website/docs/admin_hosts_houdini.md

```diff
@@ -3,9 +3,26 @@ id: admin_hosts_houdini
 title: Houdini
 sidebar_label: Houdini
 ---
+## General Settings
+### JOB Path
+you can add your studios preffered JOB Path, JOB value will be checked and updated on file save and open.
+Disableing this option will effectivly turn off this feature.
```

Comment:
```suggestion
The Houdini `$JOB` path can be customized through project settings with a dynamic path that will be updated on context changes, e.g. when switching to another asset or task.

Disabling this feature will leave `$JOB` var unmanaged and thus no context update changes will occur.
```

Might it be good to also link to the Houdini documentation as to what the JOB variable is intended to be for Houdini itself?

Reply:
okay, no more spell typos (I hope), I'm going to install vs code spell checker.